dockerfiles/ubuntu_20.04

```diff
@@ -0,0 +1,84 @@
+ARG BASE_IMAGE
+FROM ubuntu:20.04
+#FROM ${BASE_IMAGE}
+
+RUN useradd --user-group --create-home --no-log-init --shell /bin/bash tarantool
+
+# An ARG instruction goes out of scope at the end of the build
+# stage where it was defined. To use an arg in multiple stages,
+# each stage must include the ARG instruction
+ARG TNT_VER
+ARG NPROC
+ENV TARANTOOL_VERSION=${TNT_VER} \
+    LUAROCK_VSHARD_VERSION=0.1.18 \
+    LUAROCK_AVRO_SCHEMA_VERSION=3.0.6 \
+    LUAROCK_EXPERATIOND_VERSION=1.1.1 \
+    LUAROCK_QUEUE_VERSION=1.1.0 \
+    LUAROCK_CONNPOOL_VERSION=1.1.1 \
+    LUAROCK_HTTP_VERSION=1.2.0 \
+    LUAROCK_MEMCACHED_VERSION=1.0.1 \
+    LUAROCK_METRICS_VERSION=0.12.0 \
+    LUAROCK_TARANTOOL_PG_VERSION=2.0.2 \
+    LUAROCK_TARANTOOL_MYSQL_VERSION=2.1.0 \
+    LUAROCK_TARANTOOL_MQTT_VERSION=1.5.1 \
+    LUAROCK_TARANTOOL_GIS_VERSION=1.0.1 \
+    LUAROCK_TARANTOOL_PROMETHEUS_VERSION=1.0.4 \
+    LUAROCK_TARANTOOL_GPERFTOOLS_VERSION=1.0.1
+
+ARG ENABLE_BUNDLED_LIBYAML \
+    LUAJIT_DISABLE_SYSPROF \
+    LUAJIT_ENABLE_GC64
+
+RUN set -x && apt update && apt -y --no-install-recommends install libgoogle-perftools4 \
+    libgoogle-perftools4 \
+    deborphan \
+    unzip \
```

Comment:
I would remove packages like unzip and curl with dependencies after installation to reduce image size

Reply:
please check other redundant packages which can be removed for the same reason